docs/changelog.rst

```diff
@@ -11,6 +11,15 @@ with advance notice in the **Deprecations** section of releases.
 
 .. towncrier release notes start
 
+v3.8.4 (2019-03-30)
+-------------------
```

Comment:
eh, not here, see https://github.com/tox-dev/tox/blob/master/docs/changelog/examples.rst

Reply:
Oops. fixed it